bin/common.sh

```diff
@@ -149,7 +159,7 @@ function run() {
   wait_for_resources_to_be_ready "kafkachannels.messaging.knative.dev" || return $?
   wait_for_resources_to_be_ready "kafkasources.sources.knative.dev" || return $?
 
-  wait_for_workloads_to_be_running || return $?
+  wait_until_pods_running "knative-eventing" || return $?
   wait_until_pods_running "${TEST_CASE_NAMESPACE}" || return $?
```

Comment:
Verifying Kafka has the benefit that we are sure we're testing against a really running cluster event after creating a bunch of topics

Reply:
The `run.sh` script, which is used by the other scripts used in the CI, is like this:

```
apply_manifests
run
```

I didn't remove the `wait_until_pods_running "kafka" || return $?` in `apply_manifests`, so, we still check if Kafka stuff is up and running.

I removed the check for Kafka stuff in `run` function only. This would not have any effect for the CI scenario.

It would only have an effect if I run the `run` function separately from `apply_manifests`. As I explained above in other comments, I want to be able to use the `run` function separately.

Reply:
Sounds good